spec/views/groups/index.html.haml_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+RSpec.describe "groups/index", type: :view do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.